doc/questions.md

```diff
@@ -0,0 +1,55 @@
+# ros-get questions
+
+- *1. where do these packages come from??*
+  - Would be the ros-distro story, but needs a good place in the manual.
+  - perhaps an example and a eference to the REP.
+
+- *2. which branch?*
+  - ``ros-get update`` does what wrt branches?
+  - should it do other things with branches too? (make them, remove them, switch between them, push them, fetch/pull them, etc)
+  - *3. are all branches updated?* Assuming you can have some repos using a different branch, are these updated too, or is that configurable, or?
+
+- *4. is this true?*
+  - ``ros-get list`` seems to list the top-level packages?
```

Comment:
That is true, maybe that functionality should be moved to an `--installed` argument. By default list ALL packages?